MBBSEmu.Tests/ExportedModules/Majorbbs/file_Tests.cs

```diff
@@ -622,6 +622,26 @@ public void fputs_fseek_EOF_file()
             Assert.Equal(0, fclose(filep));
         }
 
+        [Fact]
+        public void f_read_InvalidStream_Throw()
+        {
+            //Reset State
+            Reset();
+
+            //Pass empty pointer
+            Assert.Throws<FileNotFoundException>(() => fread(new FarPtr(), 0, 0, new FarPtr()));
```

Comment:
not a big deal, but `new FarPtr() == FarPtr.Empty`, interchangable and saves you having to allocate a new FarPtr